src/ticTacToe/Main.java

```diff
@@ -2,6 +2,8 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.print("Hello world!");
+        System.out.println("X O X ");
```

Comment:
It looks good but you do not need an extra space at the end of your line: "X O X " -> "X O X"